SPECS/yp-tools/yp-tools.spec

```diff
@@ -3,15 +3,16 @@ Distribution:   Mariner
 Summary: NIS (or YP) client programs
 Name: yp-tools
 Version: 4.2.3
-Release: 9%{?dist}
+Release: 14%{?dist}
 License: GPLv2
 Source: https://github.com/thkukuk/yp-tools/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Patch1: yp-tools-2.12-hash.patch
 Patch2: yp-tools-2.12-crypt.patch
 Patch3: yp-tools-2.12-adjunct.patch
 Patch4: yp-tools-4.2.2-strict-prototypes.patch
 Patch5: yp-tools-4.2.3-yppasswd.patch
-Url: https://github.com/thkukuk/yp-tools
+Url: http://www.linux-nis.org/nis/yp-tools/index.html
```

Comment:
The new URL doesn't work. Could you double-check it's correct and if not, revert to the GitHub one?

Reply:
The new URL is broken, reverting to GitHub link